package.json

```diff
@@ -905,7 +905,7 @@
     "adm-zip": "^0.5.9",
     "ajv": "^8.12.0",
     "ansi-regex": "^6.0.1",
-    "antlr4ts": "^0.5.0-alpha.3",
+    "antlr4": "^4.13.1-patch-1",
```

Comment:
This is the antlr4[ javascript runtime](https://github.com/antlr/antlr4/blob/dev/doc/typescript-target.md) and should not be confused with the java cli used to generate the lexer and parser for defined languages.